static/index.html

```diff
@@ -63,11 +63,44 @@
-        <p>Since 2014, Drupalcamp Cebu is brought to you by the <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu User Group.</a> <a href="#">Terms of use.</a> <a href="#">Code of conduct.</a></p>
-        <p>Made with <i class="fa fa-heart" aria-hidden="true"></i> using Drupal 8. <a href="#">Access the source code.</a></p>
+    <div class="footer-top-wrapper">
+      <div class="container">
+        <div class="col-md-12">
+          <div class="block-footer-left col-md-4">
+            <h2>Drupal in Cebu</h2>
+            <p>This camp is brought to you by <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu User Group</a></p>
+            <p>Access the source code of this website on Github. This website is all open source</p>
+            <p>Drupal meetups every last Wednesday of the month, find us on <a href="https://www.meetup.com/drupal-ph/?_cookie-check=wCreirVnK3P6nhQS">meetup.com</a></p>
+            <p class="button-link">
+              <a class="button-link text-display-large" href="https://www.meetup.com/drupal-ph/?_cookie-check=wCreirVnK3P6nhQS" target="_blank">
+                Drupal Meetups every Month
+              </a>
+            </p>
+          </div>
+          <div class="block-footer-center col-md-4">
+            <h2>Drupalcebu on Twitter</h2>
+            <p><i class="fa fa-twitter" aria-hidden="true"></i> <a href="#">@DrupalCampCebu</a> We are now open for sponsorship, contact us if you would like to promote your business</p>
+            <p><i class="fa fa-twitter" aria-hidden="true"></i> <a href="#">@DrupalCampCebu</a> We are now open for sponsorship, contact us if you would like to promote your business</p>
+            <p><i class="fa fa-twitter" aria-hidden="true"></i> <a href="#">@DrupalCampCebu</a> We are now open for sponsorship, contact us if you would like to promote your business</p>
+          </div>
+          <div class="block-footer-right col-md-4">
+            <h2>Hosted by CIT-U</h2>
+            <p>Cebu Institute of Technology - University Address. N. Bacalso Avenue, Cebu City</p>
+            <p class="cit-details">Telephone No. 2617741</p>
+            <p class="cit-details">Fax No. 2617743</p>
+            <p class="cit-details">Email: info@cit.edu</p>
+            <p class="cit-details">Web: http://www.cit.edu</p>
+          </div>
+        </div>
+      </div> 
+    </div>
+    <div class="footer-bottom-wrapper">
+      <div class="container">
+        <div class="col-md-12">
+          <div class="camp-logo-footer"><img src="images/drupalcebu_puso.svg" /></div>
+          <p>Since 2014, Drupalcamp Cebu is brought to you by the <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu User Group.</a> <a href="#">Terms of use.</a> <a href="#">Code of conduct.</a></p>
+          <p>Made with <i class="fa fa-heart" aria-hidden="true"></i> using Drupal 8. <a href="#">Access the source code.</a></p>
```

Comment:
Link to source code is https://github.com/Luukyb/drupalcampcebu2017, and open as a new tab.

Reply:
Done.